src/browser/views/error.handlebars

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no"
+    />
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="style-src 'self'; manifest-src 'self'; img-src 'self' data:; font-src 'self' data:;"
+    />
+    <title>{{ERROR_TITLE}} — Code Server</title>
+    <link rel="icon" href="{{assetPath csStaticBase '/media/favicon.ico'}}" type="image/x-icon" />
+    <link rel="manifest" href="{{assetPath base '/code-server.webmanifest'}}" crossorigin="use-credentials" />
+    <link rel="apple-touch-icon" href="{{assetPath csStaticBase '/media/pwa-icon-384.png'}}" />
+    <link href="{{assetPath csStaticBase '/register.css'}}" rel="stylesheet" />
```

Comment:
```suggestion
    <link href="{{assetPath base '/register.css'}}" rel="stylesheet" />
```

Or `<link href="{{assetPath base '/dist/register.css'}}" rel="stylesheet" />` if we end up moving the `dist` endpoint.